src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleFileGlobals.java

```diff
@@ -1170,4 +1170,29 @@ public void localPathOverride(String moduleName, String path, StarlarkThread thr
     validateModuleName(moduleName);
     context.addOverride(moduleName, new NonRegistryOverride(LocalPathRepoSpecs.create(path)));
   }
+
+  @StarlarkMethod(
+      name = "flag_alias",
+      doc =
+        """
+          Maps a command-line flag --foo to a Starlark flag --@repo//defs:foo. Bazel translates all
+          instances of $ bazel build //target --foo to $ bazel build //target --@repo//defs:foo.
+        """,
+      parameters = {
+          @Param(
+              name = "name",
+              doc = "The name of the flag.",
+              positional = true),
+          @Param(
+              name = "starlark_label",
```

Comment:
Sorry to be picky, but how about `starlark_flag`?  `starlark_` and `label` sound a bit redundant w.r.t. each other, and don't highlight the flag part.

So:
```py
flag_alias(name = "foo", starlark_flag = "//myflags:foo")
```

That's the clearest wording I a can think of, so someone seeing that line in a `MODULE.bazel` can have the quickest intuition of what it does.